samples/qa-tablayout/src/views/MainPanel.js

```diff
@@ -0,0 +1,15 @@
+import TabLayout, {Tab} from '@enact/sandstone/TabLayout';
```

Comment:
Can we add a screenshot test instead of adding this sample?

Reply:
I tried to test screenshot test instead of adding qa-tablayout, however couldn't because this issue didn't reproduce with screenshot test.

Reply:
Okay